code/game/objects/items/trash.dm

```diff
@@ -108,5 +108,7 @@
 	pixel_x = rand(-4,4)
 	pixel_y = rand(-4,4)
 
+
+// I'm pretty sure there was a reason this was empty, so imma just make it send signals. -steamport

```

Comment:
can you instead of assuming that just look at the code and check why its not calling parent?

just add NOBLUDGEON and it will have the same effect. Dont send signals on every subtype ever. Most of the time you can just call parent

Reply:
ah, my bad.